### PR TITLE
docs: update spanish

### PR DESCRIPTION
**Pull request by MrJmpl3**
For Hackoctober, I update the docs of spanish.

Any typo, please comment to fix.

### CODE REVIEW

**Comment by MrJmpl3** [comment]
I forgot tag you: @kevinmarrec 😃

**Comment by MrJmpl3** [comment]
I think kevin is occupied, can you see this PR @posva?

File: docs/cookbook/store.md [hunk 1]
```diff
@@ -197,16 +197,12 @@ Alternatively, you can provide your own types at the point of use.
...
+import { RootState } from '~/store'
 
 @Component
 export default class MyComponent extends Vue {
     get myThings() {
         return (this.$store.state as RootState).things
     }
-
-    mounted() {
-        const name = this.$store.getters['name'] as ReturnType<typeof getters.name>
```
Comment (kevinmarrec) [inline docs/cookbook/store.md:209]: I think the point here was to show how to `provide own types at the point of use`, why should we remove it then ?

**Review by kevinmarrec — COMMENTED** [review]
Sorry for delay, commented a little thing before we merge

**Comment by kevinmarrec** [comment]
Thanks for the PR @MrJmpl3 :)

File: docs/cookbook/store.md (hunk 1 above)
Comment (MrJmpl3) [reply docs/cookbook/store.md]: Oh, I think updated the original code instead the spanish code 😅

I will fix that.

**Review by kevinmarrec — APPROVED** [review]

**Pull request merged**